drivers/SmartThings/matter-window-covering/src/init.lua

```diff
@@ -22,6 +22,14 @@ local MatterDriver = require "st.matter.driver"
 local DEFAULT_LEVEL = 0
 local PROFILE_MATCHED = "__profile_matched"
 
+local OperationalStateEventEnum = {}
+OperationalStateEventEnum.NO_EVENT = 0x00
+OperationalStateEventEnum.CURRENT_POSITION_EVENT = 0x01
+OperationalStateEventEnum.OPERATIONAL_STATE_EVENT = 0x02
+
+local event_state = OperationalStateEventEnum.NO_EVENT
+local is_opening_closing = false
```

Comment:
would this more accurately be named `is_moving`?

Reply:
I didn't know about it. Thank you~!! I'll modify it.